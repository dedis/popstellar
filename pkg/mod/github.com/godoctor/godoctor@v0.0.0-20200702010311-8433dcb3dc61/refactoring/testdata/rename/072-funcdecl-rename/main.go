package main

func mail() { // <<<<<rename,3,6,3,10,main,pass
}
