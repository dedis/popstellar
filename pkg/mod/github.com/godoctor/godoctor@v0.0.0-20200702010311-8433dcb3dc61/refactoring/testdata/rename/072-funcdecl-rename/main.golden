package main

func main() { // <<<<<rename,3,6,3,10,main,pass
}
